pygmt/clib/session.py

```diff
@@ -922,6 +922,9 @@ def open_virtual_file(self, family, geometry, direction, data):
         direction : str
             Either ``'GMT_IN'`` or ``'GMT_OUT'`` to indicate if passing data to
             GMT or getting it out of GMT, respectively.
+            By default, GMT can modify the data you pass in. Add modifier
+            ``'GMT_IS_REFERENCE'`` to tell GMT the data are read-only, or
+            ``'GMT_IS_DUPLICATE'' to tell GMT to duplicate the data.
```

Comment:
Was thinking if we should expose `GMT_IS_DUPLICATE` as an option in PyGMT. Doesn't seem to be an obvious use case from reading https://github.com/GenericMappingTools/gmt/pull/3528, so maybe leave it for now.

Reply:
It's not clear to me when we should or need to use `GMT_IS_DUPLICATE`, but I think we can leave it for now.